src/app/public/modules/dropdown/dropdown-item.component.scss

```diff
@@ -8,7 +8,6 @@
   margin: 4px;
   min-width: 160px;
   text-align: left;
-  transition: background-color $sky-transition-time-short;
```

Comment:
I removed this because it was causing problems with setting a background color on the tab dropdown item. The animation was barely perceptible anyway.

Reply:
I always love removing animations. Simple and static!